### PR TITLE
Added cucumber task with re running support of failed tests

### PR DESCRIPTION
**Pull request by SaulCM**
Failed tests run with command: ./gradlew cucumber --featurePath "@target/failedrerun.txt"

### CODE REVIEW

**Review by GustavoZHA — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

File: src/main/resources/log4j2.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
+property.filename = logs
```
Comment (GustavoZHA) [inline src/main/resources/log4j2.properties]: If log4j.properties, delete that field

File: src/test/java/org/salesforce/hooks/IndividualScenarioHooks.java [hunk 2]
```diff
@@ -26,8 +30,12 @@ public IndividualScenarioHooks(final WebDriverManager webDriverManager) {
     }
 
     @After(value = "@CreateIndividual")
-    public void setLast() {
+    public void deleteIndividual() {
+        log.info("Delete Individual");
         IndividualListPage individualListPage = pageTransporter.navigateToIndividualListPage();
         individualListPage.deleteLastModifiedRecord();
+        String actual = individualListPage.getDeletedSuccessMessage();
+        String expected = "was deleted.";
```
Comment (GustavoZHA) [inline src/test/java/org/salesforce/hooks/IndividualScenarioHooks.java]: the assert  is not necessary in after

**Review by JorgeRCV — APPROVED** [review]

File: src/main/resources/log4j2.properties (hunk 1 above)
Comment (SaulCM) [reply src/main/resources/log4j2.properties]: Done

File: src/test/java/org/salesforce/hooks/IndividualScenarioHooks.java (hunk 2 above)
Comment (SaulCM) [reply src/test/java/org/salesforce/hooks/IndividualScenarioHooks.java]: Done

**Pull request merged**